packages/codecs/ilp/src/index.ts

```diff
@@ -215,8 +215,8 @@ export const serializeIlpReject = (json: IlpReject) => {
   return envelope.getBuffer()
 }
 
-export const deserializeIlpReject = (binary: Buffer): IlpReject => {
-  const { type, contents } = deserializeEnvelope(binary)
+export const deserializeIlpRejectLOL = (binary: Buffer): IlpReject => {
```

Comment:
LOL!

Reply:
Fixed 😄

Reply:
Happy coincidence, I was just about to link to this PR from the WM issue re: monorepo structure :)